module/cmd/peggy/cmd/eth_keys.go

```diff
@@ -0,0 +1,130 @@
+package cmd
+
+import (
+	"crypto/ecdsa"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/keys"
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/tendermint/libs/cli"
+)
+
+const flagPassphrase = "passphrase"
+
+// Commands registers a sub-tree of commands to interact with
+// local private key storage.
+func Commands(defaultNodeHome string) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "eth_keys",
```

Comment:
NIT: I really prefer typing `eth-keys` and the rest of the commands use `-`

Reply:
I basically copied the cosmos `keys add` implementation. If we switch them, we should probably do all of them in peggy?